### ISSUE TITLE
Replace CPAP & hearing aid form links on 4 pages

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#75016]

**Comment by laurwill** [comment]
Hi @kristinoletmuskat , we got this request from cartography to change URLs for the CPAP form. Could you confirm IA is okay for this change? And please add any steps/tasks I missed here to the list. Thank you!

Tagging @carlosfelix2 @wesrowe for visibility and in case we have questions.

**Comment by carlosfelix2** [comment]
@kristinoletmuskat We have a virtual redirect in place while we do the migration. Otherwise we will end up with a circular redirect. We will be doing a reverse proxy redirect after the links are changed in the content. You can check our or migration checklist for this app [in the comments in this issue](https://app.zenhub.com/workspaces/mhv-on-vagov-top-level-view-62619a987d74510018ecc546/issues/gh/department-of-veterans-affairs/va.gov-team/70757).

**Comment by carlosfelix2** [comment]
Also, do not request a redirect until we can change the React app's root URL which happens after all the content changes. Otherwise you will get a circular redirect.

**Comment by wesrowe** [comment]
@kristinoletmuskat – @sterkenburgsara noticed this morning that the links to this reordering app are not protected by a login widget. For some reason, the ordering app (/introduction) handled authentication. Sara believes that these static pages should be updated to require login before showing the link to users, per the usual pattern. 

- Do you agree?
- If so, would you like me to update the ticket body above to reflect it?

**Comment by kristinoletmuskat** [comment]
Hey @wesrowe, yes Sara is right! The sign-in widget should be on the static page, like https://www.va.gov/health-care/get-medical-records/

**Comment by kristinoletmuskat** [comment]
@laurwill could we add sign-in widget updates to this ticket for Derek to implement, or do we need a diff ticket? Also note that we will need to update both these pages:

1. CPAP: https://www.va.gov/health-care/order-cpap-supplies/
2. Hearing aids: https://www.va.gov/health-care/order-hearing-aid-batteries-and-accessories/

This was implemented incorrectly originally because we copied hearing aids, which was implemented incorrectly! Or at least before we had clear guidance.

**Comment by kristinoletmuskat** [comment]
Hey hey @wesrowe just connected with Laura, and we actually need to circle back with OCTO on these changes. We learned that we have a different (and opposite) implementation of the sign-in widget for tools vs forms, and we need to determine if we want to make a change to align these experiences, and if so how to go about that -- it will require team and dev time unfortunately, which I didn't realize. @sterkenburgsara FYI our guidance on this is actually wonkier than I thought.

**Comment by wesrowe** [comment]
Hey @kristinoletmuskat, sounds fine to me. From our perspective, the important thing is that we don't complete the move of this form to /my-health/ until the widgets are in place.

are you creating a new ticket for the sign-in widgets then? I'd like to grab that link.

**Comment by kristinoletmuskat** [comment]
Yes just made one: https://github.com/department-of-veterans-affairs/va.gov-team/issues/77499

Is there anything re: timing you want us to know? I know this is probably asap but also it's likely going to take us a little bit to get OCTO alignment.

**Comment by wesrowe** [comment]
@kristinoletmuskat, there are technical processes going on that will take a couple more weeks probably, before we'll be ready to finalize moving the app. So that buys you some time. Also, I hate to say this, but there is no congressional mandate that we move that app by a certain date.

**Comment by kristinoletmuskat** [comment]
Hey ya'll, we were able to sync on this and confirmed our guidance for sign-in widgets: it should be placed in the furthest possible location a user can access before they have to sign in. For CPAP/hearing aids, this means that once the form is moved to myhealth, the sign-in widget should be on these pages:

CPAP: https://www.va.gov/health-care/order-cpap-supplies/
Hearing aids: https://www.va.gov/health-care/order-hearing-aid-batteries-and-accessories/

We can't build the widget ourselves -- we need some dev time to make those. Is that something your team will do? Or should we ask a different team?

A note on timing: Laura is working on changes to the sign-in widget copy to be clearer. But that is going to take a little bit of time to roll out. If we want to implement this sooner we can copy the Appointments widget implementation, or we can wait to implement until Laura is done -- do ya'll have a preference?

@wesrowe @sterkenburgsara @laurwill

**Comment by wesrowe** [comment]
Hi @kristinoletmuskat, given that it's the unauthenticated experience, have you connected with Public Websites?

**Comment by kristinoletmuskat** [comment]
@wesrowe do you mean for building the widget? We have not but I can!

**Comment by wesrowe** [comment]
@kristinoletmuskat, yes, I meant asking PW about the widget implementation. Given it's in the benefit hubs, they're best positioned.

**Comment by kristinoletmuskat** [comment]
Sorry for the delay, but just followed up with Jill on public websites.

**Comment by kristinoletmuskat** [comment]
Note: this work is blocked by moving the widget to the static page I think? https://github.com/department-of-veterans-affairs/va.gov-team/issues/77499

**Comment by VCM86** [comment]
These sub-tasks are still blocked. See comment in main epic for more info.

**Comment by VCM86** [comment]
Update from Kay in this [slack thread](https://dsva.slack.com/archives/C096YQLRWHK/p1762282877566369): "we won't have a stable test environment until FY26 Q2- January- March 2026."
